mobile/src/main/java/ch/epfl/sdp/mobile/state/StatefulSettingsScreen.kt

```diff
@@ -1,16 +1,17 @@
 package ch.epfl.sdp.mobile.state
 
-import androidx.compose.runtime.Composable
-import androidx.compose.runtime.remember
+import androidx.compose.runtime.*
 import androidx.compose.ui.Modifier
 import ch.epfl.sdp.mobile.application.Profile.Color
 import ch.epfl.sdp.mobile.application.authentication.AuthenticatedUser
 import ch.epfl.sdp.mobile.ui.setting.SettingScreenState
 import ch.epfl.sdp.mobile.ui.setting.SettingsScreen
 import ch.epfl.sdp.mobile.ui.social.ChessMatch
 
-class AuthenticatedUserProfileScreenState(private val user: AuthenticatedUser) :
-    SettingScreenState {
+class SettingsScreenState(
```

Comment:
It's strange having `SettingsScreenState` which implements `SettingsScreenState`. What about naming the implementation `SnapshotSettingsScreenState` ?